src/main/java/de/tum/in/www1/artemis/web/rest/dto/BuildTriggerDTO.java

```diff
@@ -0,0 +1,19 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import java.util.List;
+
+/**
+ * DTO used for specificing participationIds of programming exercises for which the build should be triggered.
+ */
+public class BuildTriggerDTO {
```

Comment:
Do we really need this, if it is just a wrapper for a List of Longs? Could we not just send a `List<Long>` object instead?
If we really want to keep this and someone else wants to reuse this: Should we then not rename this to ParticipationListDTO?

Reply:
I was not aware that it is possible to use a list/set here directly. I'll remove the DTO.